### DISCUSSION TITLE
Execute OCI hooks as non root

### DISCUSSION THREAD

**Discussion by dominicmeyer** [containers/podman#22282] [Q&A]
Hi,
I wanna create a local OCI hook, but I noticed they always seem to be run as root user.
I tested this by just creating a hook, which points to a script:
```bash
whoami > /tmp/test.txt
```
I wanna access the other running containers through the podman cli, but because the hook is run as root I can't access them.
Is there right now a way of changing this behaviour?

Of course I could use 
```
su otheruser
```
But then I have to hardcode my password into this script.

Sorry if I might have missed something in the Documentation, but I couldn't find something like this.

**Comment by eriksjolund** [comment]
Just brainstorming here. This a half-baked idea:
Maybe some day in the future it will be possible
 to run rootless podman in a systemd system service that uses the `User=` directive.
Things that need root privileges to run could use the plus syntax (`+`) in the path, for example `ExecStartPre=+/some/command/to/run/as/root` or   `ExecStartPost=+/some/command/to/run/as/root`

* https://github.com/containers/podman/discussions/20573
(The feature is currently not supported as I understand it)
